contents/docs/llm-analytics/_snippets/llms-installation-platforms.tsx

```diff
@@ -33,6 +33,11 @@ const LLMInstallationPlatforms = () => {
             url: '/docs/llm-analytics/installation/openrouter',
             image: 'https://res.cloudinary.com/dmukukwp6/image/upload/openrouterai_8260e8b011.png',
         },
+        {
+            label: 'LiteLLM',
+            url: '', // TODO: Add LiteLLM link
+            image: '/static/images/litellm-logo.png',
```

Comment:
Here's how you can upload images to Cloudinary FYI: https://posthog.com/handbook/engineering/posthog-com/assets